script/yaml2json.py

```diff
@@ -11,7 +11,9 @@
 
 
 def main():
-    print(json.dumps(yaml.load(sys.stdin, Loader=yaml.Loader), sort_keys=True,
+    x = yaml.load(sys.stdin, Loader=yaml.Loader)
+    del x["_ANCHORS"]
+    print(json.dumps(x, sort_keys=True,
```

Comment:
Nicely done.

If you're amenable to further improvements, I'd be keen to chat.  I was considering replacing these helper scripts with gomplate and/or jcp/jq.